x-pack/plugins/observability/public/hooks/use_get_cases_user_permissions.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useEffect, useState } from 'react';
+import { useKibana } from '../utils/kibana_react';
+
+export interface UseGetUserCasesPermissions {
```

Comment:
The file name is `use_get_cases_user_permissions` but the function is `useGetUserCasesPermissions`. Could these be updated to match?